tests/end-to-end/log-manager/compaction-audit-sc.gen.bats

```diff
@@ -85,11 +85,15 @@ setup() {
   #       is still no guarantee.
   test_run_cronjob "${CRONJOB_NAME}" 1200
 
-  object_storage.is_compacted "${BUCKET}" "${PREFIX}"
+  local -r cutoff="$(_get_latest_job_start_time)"
+
+  object_storage.is_compacted "${BUCKET}" "${PREFIX}" "${cutoff}"
 }
 
-@test "audit-sc - log-manager compaction job runs fine when no logs needs compaction" {
-  object_storage.is_compacted "${BUCKET}" "${PREFIX}"
+@test "audit-sc - log-manager compaction job runs fine when no logs need compaction" {
+  local -r cutoff="$(_get_latest_job_start_time)"
+
+  object_storage.is_compacted "${BUCKET}" "${PREFIX}" "${cutoff}"
 
   test_run_cronjob "${CRONJOB_NAME}" 120
 }
```

Comment:
I think what caused the setup initially was this test. How do we know that there is "no logs that need compaction" when it runs? I think this is no longer can't be guaranteed.

It was part of the initial issue description but maybe I should have questioned it rather than trying to fit it in the e2e suite.

I would argue that this is test too low level and should be moved to the integration test instead.